src/pages/MerchandiseList/MerchandiseList.tsx

```diff
@@ -0,0 +1,57 @@
+import {
+    Flex,
+    Divider,
+    Text,
+    Select,
+    Center,
+    Show,
+    Hide,
+    Heading,
+} from "@chakra-ui/react";
+
+import Card from "./Card";
+import Footer from "../../components/Footer";
+import Header from "../../components/Header";
+
+const MerchandiseList = () => {
+    return (
+        <Flex direction='column'>
+            <Header />
+            <Text textAlign='center' textStyle={['h6', 'h5']} textColor='primary.600' mt={5} mb={5}>SCSE Merchandise</Text>
+            <Select bgColor={['white', 'gray.100']} w={125} h={8} alignSelf='center' borderRadius={20} placeholder='Product Type' size='xs' />
+            <Flex justifyContent='space-between' mt={5} mb={5} alignItems='center'>
+                <Heading ml={[5, 16]} fontSize={['md', '2xl']} textColor={['primary.600', 'black']}>New Drop</Heading>
+             
+                <Flex mr={[5, 20]} alignItems='center'>
+                    <Text mr={[3, 5]} fontSize={['xs', 'xl']} textColor='primary.600'>Sort By:</Text>
+                    
+                    <Hide above='sm'>
+                        <Select w={[100, 180]} placeholder='Date - New to Old' size='xs' borderRadius={6}/>
+                    </Hide>
+                    <Show above='sm'>
+                        <Select w={[100, 180]} placeholder='Date - New to Old' size='sm' borderRadius={6}/>
+                    </Show>
```

Comment:
You don't need to use hide/show components here. Instead let's set the `size` prop of \<Select\/\> to a variable. i.e.

```tsx
import { useBreakpointValue } from '@chakra-ui/react'

const selectSize = useBreakpointValue({ base: 'xs', md: 'sm' })
// change the base, md an other values accordingly

size={selectSize}
```